src/components/content-tab-import-export.tsx

```diff
@@ -1,6 +1,132 @@
+import { createInterpolateElement } from '@wordpress/element';
+import { sprintf } from '@wordpress/i18n';
+import { Icon, download } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import { useRef } from 'react';
+import { useConfirmationDialog } from '../hooks/use-confirmation-dialog';
+import { useDragAndDropFile } from '../hooks/use-drag-and-drop-file';
+import { useSiteDetails } from '../hooks/use-site-details';
+import { cx } from '../lib/cx';
+import { getIpcApi } from '../lib/get-ipc-api';
+import { ImportState } from '../lib/import-export/import/types';
+import Button from './button';
+import { ProgressBarWithAutoIncrement } from './progress-bar';
+
 interface ContentTabImportExportProps {
 	selectedSite: SiteDetails;
 }
-export function ContentTabImportExport( _props: ContentTabImportExportProps ) {
-	return null;
+
+export function ContentTabImportExport( props: ContentTabImportExportProps ) {
+	const { __ } = useI18n();
+	const { importFile, updateSite, startServer } = useSiteDetails();
+	const importConfirmation = useConfirmationDialog( {
+		message: sprintf( __( 'Overwite %s?' ), props.selectedSite.name ),
+		checkboxLabel: __( 'Don’t show this message again' ),
+		detail: __( 'Importing a backup will replace the existing files and database for your site.' ),
+		confirmButtonLabel: __( 'Import' ),
+		localStorageKey: 'dontShowImportConfirmation',
+	} );
+
+	const { dropRef, isDraggingOver } = useDragAndDropFile< HTMLDivElement >( {
+		onFileDrop: ( file: File ) => {
+			importConfirmation( () => importFile( file, props.selectedSite ) );
+		},
+	} );
+	const inputFileRef = useRef< HTMLInputElement >( null );
+	const openFileSelector = async () => {
+		inputFileRef.current?.click();
+	};
+	const onFileSelected = async ( e: React.ChangeEvent< HTMLInputElement > ) => {
+		const file = e?.target?.files?.[ 0 ];
+		if ( ! file ) {
+			return;
+		}
+		importConfirmation( () => importFile( file, props.selectedSite ) );
+	};
+	const openSite = async () => {
+		if ( ! props.selectedSite.running ) {
+			await startServer( props.selectedSite.id );
+		}
+		getIpcApi().openSiteURL( props.selectedSite.id );
+	};
+	const clearImportState = () =>
+		updateSite( { ...props.selectedSite, importState: ImportState.Initial } );
+	return (
+		<div className="p-8 flex flex-col justify-between gap-8">
+			<div className="flex flex-col w-full">
+				<div className="a8c-subtitle-small mb-1">{ __( 'Import' ) }</div>
+				<div className="text-a8c-gray-70 a8c-body mb-4">
+					{ createInterpolateElement(
+						__( 'Import a Jetpack backup or a .sql database file. <a>Learn more.</a>' ),
```

Comment:
Nitpick: If the component we add to Learn more is a `Button`, for consistency, probably the HTML tag that we could use could be `button`.

Reply:
Nitpick: I noticed the button includes the dot, I presume the button should only contain the text Learn More, without any symbols.

```suggestion
						__( 'Import a Jetpack backup or a .sql database file. <a>Learn more</a>.' ),
```

Reply:
Changed to use button element, and moved the dot out: https://github.com/Automattic/studio/pull/369/commits/09cc3780fae2754fb249483c97e110e0ea61692b

Reply:
I think as per design, we should not need the dot at all:

<img width="464" alt="Screenshot 2024-07-23 at 10 53 03 AM" src="https://github.com/user-attachments/assets/749507e1-322a-445b-b7aa-55050c70c673">

Reply:
Done https://github.com/Automattic/studio/pull/369/commits/9cb451790240b22f9b974cab6c64a2b995987cdd